app/src/main/java/com/example/graburger/Adapter.java

```diff
@@ -47,36 +46,24 @@ public boolean isViewFromObject(@NonNull View view, @NonNull Object object) {
     @NonNull
     @Override
     public Object instantiateItem(@NonNull ViewGroup container, final int position) {
-        layoutInflater = LayoutInflater.from(context);
+        LayoutInflater layoutInflater = LayoutInflater.from(context);
         View view = layoutInflater.inflate(R.layout.card_item, container, false);
 
         ImageView imageView;
-//        TextView title, desc;
+        TextView desc;
 
         imageView = view.findViewById(R.id.image);
-//        title = view.findViewById(R.id.title);
-//        desc = view.findViewById(R.id.desc);
+        desc = view.findViewById(R.id.desc);
 
         imageView.setImageResource(models.get(position).getImage());
-//        title.setText(models.get(position).getTitle());
```

Comment:
All these is the comment is what we need to add text to the orderListCarousel, right?

Reply:
exactly :)